src/radical/pilot/agent/lm/orte.py

```diff
@@ -180,9 +184,12 @@ def lrms_shutdown_hook(cls, name, cfg, lrms, lm_info, logger):
                 orterun = ru.which('orterun')
                 if not orterun:
                     raise Exception("Couldn't find orterun")
-                subprocess.Popen([orterun, "--hnp", lm_info['dvm_uri'], "--terminate"])
+                ru.sh_callout('%s --hnp %s --terminate' 
```

Comment:
Is this relevant to this PR?

Reply:
I hear you, good question, and I assume a matter of taste.  I tend to try to improve code pieces where there are obvious legacy things missed by previous refactoring and stuff as long as it is as small as here (single line).  If that affects more code and convolutes the PR, it should got into a separate PR indeed.

Happy to jump through the exercise to move it out.  To answer explicitly: no, that change is cosmetic, and not related to the PR otherwise.